src/app/shared/stache/modules/video/video.component.html

```diff
@@ -0,0 +1,3 @@
+<div class="stache-video">
+  <iframe class="stache-video-embed" [src]="src" frameborder="0" allowfullscreen></iframe>
```

Comment:
Small change, but can we use `src="{{videoSource}}"` so that the consumer is not confused about the input? `src` is owned by `iframe`, and isn't an angular input property.